robot/EDA/resources/EDA.py

```diff
@@ -381,3 +424,187 @@ def get_namespace_prefix(self, name):
+        """ Converts the given datetime to UTC timezone
+            my_time should be in the format %Y-%m-%d %H:%M:%S
+        """
+        my_time_format = datetime.datetime.strptime(my_time, "%Y-%m-%d %H:%M:%S.%f")
+        my_time_local = pytz.timezone("America/Los_Angeles").localize(my_time_format, is_dst=None)
+
+        my_time_utc = my_time_local.astimezone(pytz.utc)
+        return datetime.datetime.strftime(my_time_utc, "%Y-%m-%dT%H:%M:%S.%f")[:-3] + "Z"
+
+    def format_all(self, loc, value):
+        """ Formats the given locator with the value for all {} occurrences """
+        count = 0
+        for s in loc:
+            if s is '{':
+                count += 1
+
+        if count == 1:
+            return loc.format(value)
+        elif count == 2:
+            return loc.format(value, value)
+        elif count == 3:
+            return loc.format(value, value, value)
+
+    def get_instance_url(self, org=None):
+        """ Returns the instance url of the target salesforce org."""
+        if org is None:
+            org = self.cumulusci.org
+        else:
+            org = self.cumulusci.keychain.get_org(org)
+        return org.instance_url
+
+    def get_eda_locator(self, path, *args, **kwargs):
+        """ Returns a rendered locator string from the eda_lex_locators
+            dictionary. This can be useful if you want to use an element in
+            a different way than the built in keywords allow.
+        """
+        locator = eda_lex_locators
+        for key in path.split('.'):
+            locator = locator[key]
+        main_loc = locator.format(*args, **kwargs)
+        return main_loc
+
+    def select_navigation_tab(self, tab):
+        """ Selects navigation tab - as passed in to the function """
+        locator_menu = eda_lex_locators["navigation_menu"]
+        element_menu = self.selenium.driver.find_element_by_xpath(locator_menu)
+        locator_tab = eda_lex_locators["navigation_tab"].format(tab)
+
+        self.selenium.wait_until_page_contains_element(
+            locator_menu,
+            error="Navigation menu drop down unavailable"
+        )
+        # javascript is being used here because the usual selenium click is highly unstable for this element on MetaCI
+        self.selenium.driver.execute_script("arguments[0].click()", element_menu)
+        time.sleep(1)
+
+        # Sometimes, single click fails. Hence an additional condition to click on it again
+        if not self.check_if_element_exists(locator_tab):
+            self.selenium.driver.execute_script("arguments[0].click()", element_menu)
+            time.sleep(1)
+
+        self.selenium.capture_page_screenshot()
+        self.selenium.wait_until_page_contains_element(
+            locator_tab,
+            error=tab + " item not found as an available option in: " + locator_tab
+        )
+        self.selenium.click_element(locator_tab)
+
+
+
+    def select_tab(self, tab):
```

Comment:
Same comment as above. If all I have is the documentation and not the code, there's no way to know what `tab` should be.

Reply:
resolved.

Reply:
I updated the documentation to be more specific as to the purpose of the tab parameter